lib/tapioca/generator.rb

```diff
@@ -94,6 +96,7 @@ def existing_rbis
     sig { returns(T::Hash[String, String]) }
     def expected_rbis
       @expected_rbis ||= bundle.dependencies
+        .reject { |gem| config.exclude.include?(gem.name) }
```

Comment:
`filter_map`?

Reply:
Ideally yes, but we aim to support all the way back to 2.4 for now, so that won't work. 😞